migrations/dev-20210610-add-users-changelogViewDate-column.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    await queryInterface.addColumn('Users', 'changelogViewDate', {
+      type: DataTypes.DATE,
+      defaultValue: DataTypes.NOW,
```

Comment:
Why a `defaultValue` of NOW?

Reply:
Not strictly needed, I just added it as I thought having a default value here makes sense than a null value, but thinking again now having a null makes sense. 👍🏽